proposals/devcontainer-features-distribution.md

```diff
@@ -0,0 +1,130 @@
+# Community Contribution and Discovery of Dev Container Features
+
+This specification defines a pattern where community members and organizations can author and self-publish [dev container 'features'](./devcontainer-features.md). 
+
+Goals include:
+
+- For community authors, a "self-served" mechanism for dev container feature publishing.
+- For community members, the ability to publish features, thus allowing  [supporting tools](../docs/specs/supporting-tools.md) to implement their own discovery mechanisms.
```

Comment:
I thought publishing features would fall under community authors, and searching for published features (i.e. through the mechanisms supporting tools implement) would be fore community members.

Reply:
Perhaps I just remove the 2nd bullet point here?  I wanted to illustrate that this spec is for _publishing_ assets for distribution, which will enable additional functionality outside this specification to aid in discovery